upgrade.adoc

```diff
@@ -0,0 +1,494 @@
+. Mailstore servers 
+
+include::upgrade-process-inc.adoc[]
+
+=== Using LMDB as the Supported Back-end for On-disk Database Maps
+
+[WARNING]
+====
+Starting with _{product-abbrev} 8.5_ and later, _Postfix_ is linked to _LMDB_, the same
+back-end _{product-abbrev}_ uses with _OpenLDAP_. Prior to _{product-abbrev} 8.0_, _Postfix_ was linked
+to _Berkeley DB_.
+
+_{product-abbrev}_ has not officially supported using any _Postfix_ on-disk database
+maps prior to _ZCS 8.5_. However, these have been used through custom
+non-preserved modifications to the `postconf` configuration. These
+modifications will be lost on upgrade.
+====
+
+To restore the modifications post-upgrade, the following steps need to
+be performed:
+
+. Run postmap against the database input file to generate an LMDB
+database.
+. It will be necessary to iterate through the postconf keys that have
+`hash:/path/to/db` values and update them in LDAP to use
+`lmdb:/path/to/db` values instead.
+
+Many previously unsupported features that could be used with on-disk
+database maps are now fully supported by _{product-abbrev}_. Check if your
+customizations are correctly carried forward when
+upgrading. See https://bugzilla.zimbra.com/show_bug.cgi?id=77586[Bug
+77586].
+
+== After the Upgrade is Complete
+
+After you completed the upgrade, the following might need to be
+addressed.
+
+* Review https://wiki.zimbra.com/wiki/How_to_disable_SSLv3[How to disable SSLv3].
+* Review https://wiki.zimbra.com/wiki/Cipher_suites[Cipher suites].
+* During the upgrade process, _{product-short}_ might make a binary backup of
+  existing databases when there are major structural changes occurring
+  to the database format for ease of downgrading. Administrators will
+  want to clean these up once they have confirmed a successful
+  upgrade. For _LDAP_ servers, these backups are in
+  `/opt/zimbra/data/ldap`, and in the form of `<dbname>.prev.+++$$+++`
+  where `+++$$+++` is the process
+  ID of the upgrade script. See https://bugzilla.zimbra.com/show_bug.cgi?id=81167[Bug 81167].
+* You should run `zmldapupgrade -b 66387` after upgrading.  The
+`zimbraAllowFromAddress` attribute cannot be set for internal accounts
+ or distribution lists. Running this script will change
+ `zimbraAllowFromAddress` values to grants.
++
+[NOTE]
+====
+* This step was not included into the installer-driven upgrade due to
+  potentially long delay for sites that set `zimbraAllowFromAddres` on
+  many accounts.
+* The migration command reports how many accounts had
+  `zimbraAllowFromAddress` attribute set and how many of
+  them needed migration. One way to verify all accounts got migrated
+  is to run the command again. The total won't change, and the number
+  migrated should be 0. See
+  https://bugzilla.zimbra.com/show_bug.cgi?id=66387[Bug 66387] for
+  more information.
+====
++
+* If your self-signed SSL certificates have expired, update them. See
+  _Verify Certificates Expiration Date_ on page 14 TODO - where is this?
```

Comment:
I haven't found anything directly useful.  The Admin guide has wording on a tool which produces:
**SSL certificates expiration notification**
and also **Managing SSL Certificates for ZCS*.
If we knew which tool generated the notifications, we could probably contain that info with a pointer to the Managing section???